observability/src/test/java/io/grpc/observability/interceptors/InternalLoggingChannelInterceptorTest.java

```diff
@@ -634,8 +622,8 @@ public String authority() {
       Metadata trailers = new Metadata();
       interceptedListener.get().onClose(status, trailers);
       interceptedLoggingCall.cancel(null, null);
-      assertTrue("LogHelper should be invoked seven times equal to number of events",
-          Mockito.mockingDetails(mockLogHelper).getInvocations().size() == 7);
+      assertEquals("LogHelper should be invoked seven times equal to number of events", 7,
```

Comment:
It is quite common to use `com.google.common.truth.Truth` `assertThat` as you can see in other unit tests.

Reply:
Updated.